ipa-core/src/protocol/ipa_prf/malicious_security/prover.rs

```diff
@@ -55,60 +56,60 @@ where
         Self { u, v }
     }
 
-    pub fn compute_proof<λ: ArrayLength>(
-        &self,
-        r: F,
-    ) -> (ZeroKnowledgeProof<F, TwoNMinusOne<λ>>, ProofGenerator<F>)
+    pub fn compute_proof<λ: ArrayLength, I, J>(
```

Comment:
this computes only one share of the proof right? the other part is generated using prss? Maybe rename proof to share of proof?

Reply:
It computes the actual polynomial G(X), which is what I am calling the "proof". This is then secret shared (using PRSS). But I consider those "shares" of the "proof".